brion/plugin/compartmentReportBinary.cpp

```diff
@@ -706,23 +716,23 @@ bool CompartmentReportBinary::_parseMapping()
                 const uint64_t frameIndex =
                     j + ((info.dataOffset - _header.dataBlockOffset) /
```

Comment:
I would probably add an assert that info.dataOffset >= _header.dataBlockOffset, unless it is guaranteed somewhere else.